package.json

```diff
@@ -36,5 +36,9 @@
     "lint:spaces": "lintspaces -n -t -d spaces -l 1 -. \"src/**/*.jsp\" \"src/**/*.tag\" \"src/main/**/*.html\" \"src/**/*.xml\" \"src/**/*.json\" \"src/**/*.css\" \"src/**/*.java\" \"src/main/webapp/dev/**/*.js\" \"src/**/*.properties\" \"*.js\" \"*.yml\" \"*.gradle\" \"static-analysis/*.*ml\"",
     "lint": "npm run lint:js && npm run lint:json && npm run lint:css && npm run lint:spaces"
   },
-  "private": true
+  "private": true,
+  "dependencies": {
+    "lintspaces": "^0.6.0",
+    "npm": "^5.8.0"
+  }
```

Comment:
You shouldn't have edited this file. Probably because you run some npm commands?

Reply:
Yes I ran the static analysis on my local and I was prompted to run ```npm -i npm``` and also install ```lintspaces```.

Reply:
1. You only need to run the `npm` static analysis when you modify javascript, i.e. `*.js` files, which is not the case here.
2. You should update your npm globally i.e. with `-g` instead of `npm -i npm`
3. You shouldn't need to install `lintspaces` again, since it is listed as a development dependency. Did you run `npm install` during [setting up the environment](https://github.com/TEAMMATES/teammates/blob/master/docs/setting-up.md)?

Reply:
Yes, I did run npm install and I do have lintspaces-cli module in my node_modules directory but I get ```lintspaces is not recognized as an internal or external command``` when I run ```npm run lint```